src/legend/svgLegend.ts

```diff
@@ -359,13 +359,23 @@ module powerbi.extensibility.utils.chart.legend {
                 })
                 .style({
                     "fill": (d: LegendDataPoint) => {
-                        if (hasSelection && !d.selected)
+                        if (hasSelection && !d.selected && !data.opacityOnNotSelected)
```

Comment:
It'd be better to call `renderSelection` from `InteractiveBehavior` instead of hardcoding some fixed behavior.

Legend's behavior should be changed via `InteractiveBehavior` only. Such was is more flexible than adding additional flags